doctr/transforms/modules.py

```diff
@@ -205,20 +215,26 @@ class Contrast(NestedObject):
         >>> out = transfo(tf.random.uniform(shape=[8, 64, 64, 3], minval=0, maxval=1))
 
     Args:
-        contrast_factor: multiplicative factor to use to augment (if > 1) or reduce (if < 1) contrast
+        lower: lower bound to pick multiplicative factor (should be < 1 to reduce contrast)
+        upper: upper bound to pick multiplicative factor (should be > 1 to augment contrast)
+        p: probability to apply transformation
     """
-    def __init__(self, contrast_factor: float = 1.3) -> None:
-        self.contrast_factor = contrast_factor
+    def __init__(self, lower: float = .7, upper: float = 1.3, p: float = .5) -> None:
+        self.lower = lower
+        self.upper = upper
+        self.p = p
 
     def extra_repr(self) -> str:
-        return f"contrast_factor={self.contrast_factor}"
+        return f"lower={self.lower}, upper={self.upper}"
 
     def __call__(self, img: tf.Tensor) -> tf.Tensor:
-        return tf.image.adjust_contrast(img, contrast_factor=self.contrast_factor)
+        if random.random() < self.p:
+            return tf.image.random_contrast(img, lower=self.lower, upper=self.upper)
+        return img
```

Comment:
same here, let's add two wrappers: `OneOf` takes will apply one of the transfo in the list and `RandomApply` that will apply the transfo in the constructor with a given probability